portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,10 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+img {
```

Comment:
Conceivably you should use a class instead of changing the style for all of the `img` tags? That way if you need to add more image content, you wouldn't need to go back and change this styling.

Reply:
At the point when I inserted the image, it was the only image, so I didn't add a class. But I agree, I should definitely do that since I plan on adding more images.